.github/workflows/README.md

```diff
@@ -29,4 +29,5 @@ Please note that jobs with matrix need to have matrix element in the comment. Ex
 | [ Python PreCommit Lint ](https://github.com/apache/beam/actions/workflows/job_PreCommit_PythonLint.yml) | N/A | `Run PythonLint PreCommit` | [![.github/workflows/job_PreCommit_PythonLint.yml](https://github.com/apache/beam/actions/workflows/job_PreCommit_PythonLint.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/job_PreCommit_PythonLint.yml) |
 | [ Python PreCommit Transforms ](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python_Transforms.yml) | ['3.8','3.9','3.10','3.11'] | `Run Python_Transforms PreCommit (matrix_element)`| [![.github/workflows/job_PreCommit_Python_Transforms.yml](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python_Transforms.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python_Transforms.yml) |
 | [ Python PreCommit ](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python.yml) | ['3.8','3.9','3.10','3.11'] | `Run Python PreCommit (matrix_element)` | [![.github/workflows/job_PreCommit_Python.yml](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/job_PreCommit_Python.yml) |
+| [ PreCommit Python IT](https://github.com/apache/beam/actions/workflows/beam_PreCommit_PythonIT.yml) | ['3.8','3.11'] | `Run PythonIT PreCommit (matrix_element)` | [![.github/workflows/beam_PreCommit_PythonIT.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_PythonIT.yml/badge.svg?event=schedule)](https://github.com/apache/beam/actions/workflows/beam_PreCommit_PythonIT.yml) |
```

Comment:
The comment trigger phrase is `Run Python_Integration PreCommit` based on the list - https://github.com/apache/beam/blob/master/release/src/main/scripts/jenkins_jobs.txt#L135

Reply:
Done